src/understanding-deis/concepts.md

```diff
@@ -39,47 +65,71 @@ view logs, run admin commands and much more.
 
 ### Build Stage
 
-The [builder][] processes incoming `git push` requests and builds applications
-inside ephemeral Docker containers, resulting in a new Docker image.
+The [builder][] component processes incoming `git push deis master` requests
+manages your application packaging.
+
+If your application is using a [buildpack][] builder will launch an ephemeral
+job to extract and execute the packaging instructions. The resulting
+application artifact is stored by the platform for execution during the run
+stage.
+
+Instead, if you provide a [Dockerfile][dockerfile] builder will use the
```

Comment:
"If instead" instead of "instead, if."  This reads better if it opens with the conditional.